### ISSUE TITLE
Ruby tests failing with "uninitialized constant RethinkDB::Shim::ReqlServerCompileError" error

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#5604]
A bunch of ReQL tests have started failing on `next` with errors like this one:

```
>>> Failed polyglot/times/api.rb1.9 with exit code 1 after 0.2 sec + 0.0 sec setup (T+ 103.2 sec)

TEST FAILURE: line 115
     SOURCE:     r.time(2013, r.july, 29).to_epoch_time()
     EXPECTED:   ReqlCompileError: Expected between 4 and 7 arguments but found 3.
     RESULT:     NameError: uninitialized constant RethinkDB::Shim::ReqlServerCompileError
     BACKTRACE:
<<<<<<<<<
```

@mlucy

**Comment by danielmewes** [comment]
This patch seems to fix it:
```rb
diff --git a/drivers/ruby/lib/exc.rb b/drivers/ruby/lib/exc.rb
index 191f18f..3eb9e7d 100644
--- a/drivers/ruby/lib/exc.rb
+++ b/drivers/ruby/lib/exc.rb
@@ -10,11 +10,11 @@ module RethinkDB
   RqlOpFailedError = ReqlOpFailedError = Class.new(ReqlAvailabilityError)
   RqlOpIndeterminateError = ReqlOpIndeterminateError = Class.new(ReqlAvailabilityError)
   RqlUserError = ReqlUserError = Class.new(ReqlRuntimeError)
-  RqlPermissionError = Class.new(ReqlRuntimeError)
+  ReqlPermissionError = Class.new(ReqlRuntimeError)
 
   RqlDriverError = ReqlDriverError = Class.new(ReqlError)
   RqlAuthError = ReqlAuthError = Class.new(ReqlDriverError)
   RqlCompileError = ReqlCompileError = Class.new(ReqlError)
-  RqlServerCompileError = Class.new(ReqlCompileError)
-  RqlDriverCompileError = Class.new(ReqlCompileError)
+  ReqlServerCompileError = Class.new(ReqlCompileError)
+  ReqlDriverCompileError = Class.new(ReqlCompileError)
 end
```

**Comment by danielmewes** [comment]
Fixed in `next` 8d65c59.

**Comment by mlucy** [comment]
Sorry about that, I didn't realize that some of the newer error types weren't aliased when I did the autoreplace.
